src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -18,6 +18,9 @@
 @interface OWSDisappearingMessagesJob ()
 
 // This property should only be accessed on the serialQueue.
+@property (nonatomic, readonly) TSStorageManager *storageManager;
```

Comment:
re: comment, where is this serialQueue access being enforced? 

Also not sure why it would be necessary to constrain access to the dedicated serial queue since it's readonly and assigned in init.